WebRoot/js/zimbraMail/abook/model/ZmContact.js

```diff
@@ -2218,6 +2219,11 @@ ZmContact.NO_MAX_IMAGE_WIDTH = ZmContact.NO_MAX_IMAGE_HEIGHT = - 1;
  */
 ZmContact.prototype.getImageUrl =
 function(maxWidth, maxHeight) {
+	// ZBUG-2621: photo in GAL
```

Comment:
no need to add references of tickets in code comments, it can be easily tracked by git history and ticket number in git commit message

Reply:
removed the code comment.